projects/router/Makefile

```diff
@@ -1,2 +1,3 @@
 router: clone-router
+	docker exec -it $(shell $(GOVUK_DOCKER) ps | grep mongo-2.6 | cut -d' ' -f1) mongo --eval "rs.initiate()"
```

Comment:
We can just do `$(GOVUK_DOCKER) run mongo-2.6 mongo --eval "rs.initiate()"`.

Reply:
I don't think `$(GOVUK_DOCKER) run` starts mongodb, as `mongo` can't connect:

```
docker-compose -f [...] run mongo-2.6 mongo --eval rs.initiate()
MongoDB shell version: 2.6.12
connecting to: test
2021-06-09T12:06:32.163+0000 warning: Failed to connect to 127.0.0.1:27017, reason: errno:111 Connection refused
2021-06-09T12:06:32.163+0000 Error: couldn't connect to server 127.0.0.1:27017 (127.0.0.1), connection attempt failed at src/mongo/shell/mongo.js:146
exception: connect failed
```

How about I change it to

```
router: clone-router
	$(GOVUK_DOCKER) up -d mongo-2.6
	docker exec -it $(shell $(GOVUK_DOCKER) ps | grep mongo-2.6 | cut -d' ' -f1) mongo --eval "rs.initiate()"
	$(GOVUK_DOCKER) run $@-lite make build
```

?

Reply:
Yeah, sorry, I revised my comment ☝️